core_backend/app/routers/admin.py

```diff
@@ -23,4 +28,32 @@ async def healthcheck(
             status_code=500, content={"message": f"Failed database connection: {e}"}
         )
 
+    if ALIGN_SCORE_METHOD == "alignScore":
```

Comment:
new healthcheck if you are using alignScore

Reply:
Don't know why this comment is `outdated`, but I added a comment on this behaviour below.